benchmarks/lcms-2017-03-21/build.sh

```diff
@@ -30,6 +24,6 @@ build_lib() {
 get_git_revision https://github.com/mm2/Little-CMS.git f9d75ccef0b54c9f4167d95088d4727985133c52 SRC
 build_lib
 
-$CXX $CXXFLAGS ${SCRIPT_DIR}/cms_transform_fuzzer.cc -I BUILD/include/ BUILD/src/.libs/liblcms2.a $FUZZER_LIB -o $FUZZ_TARGET
-cp -r $SCRIPT_DIR/seeds $OUT/
-wget -qO $FUZZ_TARGET.dict https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/icc.dict
+$CXX $CXXFLAGS $SRC/cms_transform_fuzzer.cc -I BUILD/include/ BUILD/src/.libs/liblcms2.a $FUZZER_LIB -o $OUT/fuzz-target
+cp -r /opt/seeds $OUT/
```

Comment:
Why not ADD directly in $OUT in dockerfile. Same for all other similar places.

Reply:
I'm still not exactly sure what should go in build.sh and what should go the Dockerfile because I don't think it's clear who owns $OUT and when (can build.sh delete things in there? Can the fuzzer.py delete things?).
Putting things in the build.sh also ensures that fuzzers that need to build twice work correctly.
Initially the dockerfiles I made in this PR only did what was necessary to get build.sh working. I changed this a little to install apt packages since some of these needed changes anyway. But I think it's safest to leave things as is as much as possible given how much this PR changes.